remotecv/storages/redis_storage.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# thumbor imaging service
+# https://github.com/thumbor/thumbor/wiki
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/mit-license
+# Copyright (c) 2011 globo.com thumbor@googlegroups.com
+
+
+from json import loads
+
+from aioredis import Redis
+from aioredis.sentinel import Sentinel
```

Comment:
Since [v4.2.0rc1](https://github.com/redis/redis-py/releases/tag/v4.2.0rc1), `redis-py` offers ASYNC support, as they [merged](https://github.com/redis/redis-py/pull/1899) with `aioredis`. We better keep up from the get go:

```suggestion
from redis.asyncio import Redis
from redis.asyncio.sentinel import Sentinel
```